config/roles.yml

```diff
@@ -399,3 +399,9 @@ flow_framework_read_access:
     - 'cluster:admin/opensearch/flow_framework/workflow_state/get'
     - 'cluster:admin/opensearch/flow_framework/workflow_state/search'
     - 'cluster:admin/opensearch/flow_framework/workflow_step/get'
+
+# Allows users to use all query insights APIs
```

Comment:
Are we planning to ship this as part of default demo configuration setup?
Is the action `cluster:admin/insights/top_queries/*` defined as part of the static drop-down list displayed on the front-end?

nit: new-line at the end after line 407

Reply:
For now can we skip the demo configuration and revisit them later?

Reply:
anything under `config/` folder will be shipped as part of demo setup.

Reply:
I think what @DarshitChanpura is hinting at is that you'll need another pull request to update the dropdown list in Dashboards.

- Example permissions dropdown update https://github.com/opensearch-project/security-dashboards-plugin/pull/1626

Reply:
Correct. I'm resolving this conversation as this is a non-blocker for this PR with the understanding that this role will be available as a default static role as part of the security plugin bundle.